src/cli/index.js

```diff
@@ -362,7 +362,7 @@ config.init({
   binLinks: commander.binLinks,
   modulesFolder: commander.modulesFolder,
   globalFolder: commander.globalFolder,
-  cacheRootFolder: commander.cacheFolder,
+  cacheFolder: commander.cacheFolder,
```

Comment:
This is done on purpose afaik.
@arcanis we probably need to add a comment here.